src/aosm/azext_aosm/tests/latest/scenario_test_mocks/mock_input_templates/cnf_input_template.json

```diff
@@ -1,12 +1,12 @@
 {
-    "publisher_name": "nginx-publisher",
+    "publisher_name": "automated-tests-nginx-publisher",
     "publisher_resource_group_name": "{{publisher_resource_group_name}}",
     "nf_name": "nginx",
     "version": "1.0.0",
-    "acr_artifact_store_name": "nginx-nsd-acr",
-    "location": "westcentralus",
+    "acr_artifact_store_name": "nginx-acr",
+    "location": "uaenorth",
     "images":{
-        "source_registry": "{{source_registry_id}}",
+        "source_registry": "not-used-in-test-but-cannot-be-blank",
```

Comment:
I'm a bit surprised this isn't used anymore.

Reply:
I don't know how it used to work. I just know that as it was, `{{source_registry_id}}` evaluated to a blank string (or something that resulted in that), which caused us to fail validation.

I can't add an extensive comment as it's a JSON file. Hopefully the string itself is sufficient for anyone investigating in the future.